ipaclient/install/ipa_epn.py

```diff
@@ -33,7 +33,8 @@
 import time
 
 from collections import deque
-from datetime import datetime, timedelta, UTC
+from datetime import datetime, timedelta, timezone
+UTC = timezone.utc
```

Comment:
Is it better to define our own constant or would it be better to try to import Python's datetime.UTC and define our own in case the import fails?
```
try:
    from datetime import UTC
except ImportError:
    import timezone
    UTC = timezone.utc
```

I'm not a big fan of the "try to import" idiom, but it seems to be used often.

Reply:
I don't think we should be going this far. `datetime.UTC` is an alias of `datetime.timezone.utc`. The latter is not going to be removed. I only used `datetime.UTC` because it was advertised by the depreciation message (and also it is shorter sentence). As `datetime.timezone.utc` is not going to be removed, we can use it for all scenarios anyway.

Reply:
Yes, I agree, that's the reason for my question. :-)